src/components/shared/ProjectCard.tsx

```diff
@@ -39,14 +39,14 @@ export default function ProjectCard({
   const terminalVersion = getTerminalVersion(project.terminal)
 
   return (
-    <div
+    <a
       style={{
         borderRadius: radii.lg,
         cursor: 'pointer',
         overflow: 'hidden',
       }}
-      className="clickable-border"
       key={project?.handle}
+      href={`/p/${project.handle}`}
```

Comment:
**suggestion**: to be safe, I think we need to include the `#`:

```suggestion
      href={`/#/p/${project.handle}`}
```

**suggestion (non-blocking)**: it might be nice to define a shared helper function to build a project path. This way we can use it in other places.

Reply:
Thanks good call